SpotifyPluginClient.cs

```diff
@@ -255,6 +272,18 @@ public void ToggleShuffle()
             _spotifyClient.Player.SetShuffle(shuffleRequest).GetAwaiter().GetResult();
         }
 
+        public void LikeSongById(string trackId)
```

Comment:
I think that `sp like` should like the current song, but otherwise `sq like {query}` should perform a standard search, just like the regular `sp` query does. This would keep functionality in line with the rest of the plugin.

Reply:
Added in [`added searching tracks to 'like'`](https://github.com/fow5040/Flow.Launcher.Plugin.SpotifyPremium/pull/45/commits/e00946afbbeaa645ac0779ded368734099563bde)